src/util/registerApplicationCommands.js

```diff
@@ -0,0 +1,37 @@
+const logger = require('./logger');
+const { REST } = require('@discordjs/rest');
+const { Routes } = require('discord-api-types/v9');
+const config = require('../config');
+const translateCommand = require('../commands/translate');
+
+const commands = [translateCommand].map(command => command.builder.toJSON());
+
+const rest = new REST({ version: '9' }).setToken(config.botToken);
+
+const registerApplicationCommands = guildId => {
```

Comment:
Add the following check to this function:
If guildId is empty and `config.env` is not `"production"`, don't register commands globally.

ie, when the bot is running locally  during development there's probably no need for it to constantly re-register commands globally on every start up. That can be done in production

Reply:
> If `guildId` is empty and `config.env` is not `"production"`, don't register commands globally.

I understand what you want here, I have added the code to register globally once client `ready` event [here](https://github.com/gganon/habla-bot/pull/28/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556). I thought for flexibility, the function can be called anytime to register globally without it needing to be in production environment (eg. invoking `npm run register` from the terminal without causing downtime). But sure I will apply your suggestion.

Reply:
Wait, I totally missed that code. No I think your approach is much better. The check for production happens inside client.on('ready') while this function is flexible. Good stuff!

Clearly I didn't ready your updated code properly. Sorry about that.